frontend/src/pulldasher/sort.ts

```diff
@@ -12,6 +12,26 @@ export function defaultCompare(a: Pull, b: Pull): number {
    );
 }
 
+export function QACompare(a: Pull, b: Pull): number {
+   return (
+      // Pulls I'm QAing above those I'm not
+      compareBool(isQAingByMe(a), isQAingByMe(b)) ||
+      // Pulls with no external_block above those with external_block
+      compareBool(!a.getLabel('external_block'), !b.getLabel('external_block')) ||
+      // Pulls with no QAing label above those with QAing
+      compareBool(!a.getLabel('QAing'), !b.getLabel('QAing')) ||
+      // Pulls with CR completed above those that need more
+      compareBool(a.isCrDone(), b.isCrDone()) ||
+      // Older pulls before younger pulls
+      a.created_at.localeCompare(b.created_at)
+   );
+}
+
+function isQAingByMe(pull: Pull): boolean {
+   const label = pull.getLabel('QAing');
+   return !!(label && label.user == getUser());
```

Comment:
Since you are already using `!!` can't we combine that with an optional chain to remove the first check on whether `label` is defined?

```
return !!(label?.user == getUser());
```